sfa_dash/templates/forms/base/creation_form.html

```diff
@@ -7,7 +7,15 @@
 {{ metadata | safe }}
 {% include "sections/notifications.html" %}
 {% if form_title %}<h3>{{ form_title }}</h3> {% endif %}
-<form action="{% if data_type == 'site' %} {{ url_for('forms.create_' + data_type) }}{% else %}{{ url_for('forms.create_' + data_type, uuid=site_metadata['site_id']) }} {% endif %}" method="post" enctype='{{ form_enc }}' id="{{ data_type }}-form">
+{% if data_type in ['site', 'aggregate'] %}
+<form action="{{ url_for('forms.create_' + data_type) }}" method="post" enctype='{{ form_enc }}' id="{{ data_type }}-form">
+{% elif site_metadata is defined %}
+<form action="{{ url_for('forms.create_' + data_type, uuid=site_metadata['site_id']) }}" method="post" enctype='{{ form_enc }}' id="{{ data_type }}-form">
+  <input type="hidden" name="site_id" value="{{ site_metadata['site_id'] }}">
```

Comment:
similar to the other comment. Is there a way site_metadata is not defined, but the site is valid and one could still create a forecast/obs for it?